src/lib/es5.d.ts

```diff
@@ -1582,7 +1582,7 @@ type Extract<T, U> = T extends U ? T : never;
 /**
  * Construct a type with the properties of T except for those in type K.
  */
-type Omit<T, K extends keyof any> = Pick<T, Exclude<keyof T, K>>;
+type Omit<T, K extends keyof any> = { [P in keyof T as Exclude<P, K>]: T[P]; }

```

Comment:
This should probably also be multi‑line:
```suggestion
type Omit<T, K extends keyof any> = {
    [P in keyof T as Exclude<P, K>]: T[P];
};
```